src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+@use "../utils/_variables" as *;
+
+.main {
+  display: flex;
+  max-width: 944px;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding-top: 50px;
+  margin: 0 auto;
+}
+
+.card {
+  background: $clear_white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &--position {
+    display: flex;
+    flex-direction: column;
+    padding: 32px 16px 16px;
+    max-width: 200px;
+    margin-bottom: 46px;
+  }
```

Comment:
if you do it to avoid assigning margins to BEM-blocks — assign cards to their parent (e.g. `catalog__card`) and use structural pseudo-classes
other styles, except margins, can go in `card`

Reply:
yep